perl/lib/BarnOwl/Message.pm

```diff
@@ -116,6 +118,173 @@ sub serialize {
     return $s;
 }
 
+=head2 log MESSAGE
+
+Returns the text that should be written to a file to log C<MESSAGE>.
+
+=cut
+
+sub log {
+    my ($m) = @_;
+    return $m->log_header . "\n\n" . $m->log_body . "\n\n";
+}
+
+=head2 log_header MESSAGE
+
+Returns the header of the message, for logging purposes.
+If you override L<BarnOwl::Message::log>, this method is not called.
+
+=cut
+
+sub log_header {
+    my ($m) = @_;
+    my $sender = $m->sender;
+    my $recipient = $m->recipient;
+    my $timestr = $m->time;
+    return "From: <$sender> To: <$recipient>\n"
```

Comment:
What do you mean by "Should"?  This is the format that was already present in `owl_log_generic`.  Whether or not that format is "good" is a different matter.